packages/reporting/src/components/report-list.tsx

```diff
@@ -36,18 +36,18 @@ export enum Reports {
  * Report dictionary with it associated component
  */
 
-export const ReportContainer: Record<string, React.ReactElement> = {
-  [Reports.queryAllLogs]: <AllLogsReport />,
-  [Reports.showDispenserStateReport]: <DispenserStateReportConfig />,
-  [Reports.showDoorStateReport]: <DoorStateReportConfig />,
-  [Reports.showFleetStateReport]: <FleetStateReportConfig />,
-  [Reports.showHealthReport]: <HealthReportConfig />,
-  [Reports.showIngestorStateReport]: <IngestorStateReportConfig />,
-  [Reports.showLiftStateReport]: <LiftStateReportConfig />,
-  [Reports.showLoginsReport]: <UserLoginReportConfig />,
-  [Reports.showLogoutsReport]: <UserLogoutReportConfig />,
-  [Reports.showLoginFailuresReport]: <UserLoginFailureReportConfig />,
-  [Reports.showTasksReport]: <TaskSummaryReportConfig />,
+export const ReportContainer: Record<string, (props: ConfigProps) => JSX.Element> = {
```

Comment:
Just to clarify my understanding:  We should remove the object that associates `currentReport` with the components and instead shift them into the switch statement in `report-dashboard.tsx` so that it is easier for us to track if the component is in use?